nodeup/pkg/model/kube_apiserver.go

```diff
@@ -18,10 +18,11 @@ package model
 
 import (
 	"fmt"
-	"k8s.io/kops/util/pkg/proxy"
 	"path/filepath"
 	"strings"
 
+	"k8s.io/kops/util/pkg/proxy"
```

Comment:
So I think the rule of thumb is 
```
(
   stdlib

   local packages

   externals
)
```

Reply:
Cleaned up and added it to the tests so it is picked up on all PRs